### ISSUE TITLE
Query builder admin screen

### ISSUE THREAD

**Issue by bezbeli** [CrocoBlock/suggestions#6784]
Query list is hard to follow without any kind of sorting or filter, only way is to use browser search. Would be great to have filter or sort like on Smart filter admin.

**Comment by oceandiveloper** [comment]
+1 this is really getting hard to maintain if you have dozens of queries in the query builder.

**Comment by hartmanncreate** [comment]
+1 yes, I can get really overwhelming whan have many queries.

**Comment by Crocoblock** [comment]
Was added and released in JetEngine 3.4.0
